routes/web.php

```diff
@@ -30,10 +28,10 @@
     });
 });
 
-Route::get('local-login', 'Auth\AuthController@localLogin');
-Route::get('auth', 'Auth\AuthController@authenticate');
-Route::get('auth/callback', 'Auth\AuthController@handleTwitterCallback');
-Route::get('auth/logout', 'Auth\AuthController@getLogout');
+Route::get('local-login', 'Auth\LoginController@localLogin');
```

Comment:
I feel like this being renamed to the LoginController doesn't make sense in our context. I'd rename it back to AuthController.